nova_plugin/server.py

```diff
@@ -253,7 +253,12 @@ def rename(name):
     ctx.logger.debug(
         "server.create() server before transformations: {0}".format(server))
 
-    _handle_image_or_flavor(server, nova_client, 'image')
+    if 'block_device_mapping' in server:
```

Comment:
Currently the plugin supports passing a dict directly to the create API by populating the server property or using the create operation's "args" input.. so if you relate to "block_device_mapping", "block_device_mapping_v2" should be treated as well...